jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialManager.kt

```diff
@@ -35,7 +35,8 @@ abstract class CredentialManager : SimpleModificationTracker() {
     fun getAwsCredentialProvider(providerId: ToolkitCredentialsIdentifier, region: AwsRegion): ToolkitCredentialsProvider =
         ToolkitCredentialsProvider(providerId, AwsCredentialProviderProxy(providerId, region))
 
-    fun getCredentialIdentifiers(): List<ToolkitCredentialsIdentifier> = providerIds.values.toList()
+    fun getCredentialIdentifiers(): List<ToolkitCredentialsIdentifier> = providerIds.values
+        .sortedWith(compareBy(String.CASE_INSENSITIVE_ORDER) { it.displayName })
```

Comment:
seems worth using natural order, even if `foo-100` is a antipattern.

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.comparisons/natural-order.html

Reply:
Ah, thats the method I wanted

Reply:
Nvm

Reply:
??

Reply:
I'm going to need more than that Kyle :P

Reply:
The natural order isnt number aware, see: https://pl.kotl.in/LLWk7erDF
CASE_INSENSITIVE is already a naturalOrder sort but just case insensitive.